test/llvm_autogenerated/invoke_wrapper.s

```diff
@@ -0,0 +1,201 @@
+	.text
```

Comment:
Another option might be just to handwrite a test. IIUC we should just be able to copy the call instructions (and give them dummy local args) and then the test would be a lot smaller and simpler.